packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -37,8 +39,10 @@ export default (siteVars): ChatMessageVariables => ({
   offset: pxToRem(100),
   padding: pxToRem(16),
   authorMarginRight: pxToRem(12),
-  authorFontWeight: siteVars.fontWeightBold,
+  authorColor: siteVars.gray02,
```

Comment:
Will be `gray[500]` with new color palette

Reply:
I added comments for these values